src/Sportalytics.Feed.Persistence.MongoDB/DependencyInjection.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization;
+using MongoDB.Bson.Serialization.Serializers;
+using MongoDB.Driver;
+using MongoDB.Driver.Core.Configuration;
+using Sportalytics.Feed.Domain.Entities;
+using Sportalytics.Feed.Domain.Interfaces;
+using Sportalytics.Feed.Persistence.MongoDB.Core;
+using Sportalytics.Feed.Persistence.MongoDB.Interfaces;
+using Sportalytics.Feed.Persistence.MongoDB.Settings;
+
+namespace Sportalytics.Feed.Persistence.MongoDB;
+
+public static class DependencyInjection
+{
+    public static IServiceCollection AddMongo(this IServiceCollection services, IConfiguration configuration)
+    {
+        BsonSerializer.RegisterSerializer(new GuidSerializer(BsonType.String));
+        BsonSerializer.RegisterSerializer(new DateTimeOffsetSerializer(BsonType.String));
+
+        var mongoDbSettings = new MongoDbSettings();
+        configuration.GetSection(nameof(MongoDbSettings)).Bind(mongoDbSettings);
+
+        ArgumentNullException.ThrowIfNull(mongoDbSettings.Host);
+        ArgumentNullException.ThrowIfNull(mongoDbSettings.Port);
+        ArgumentNullException.ThrowIfNull(mongoDbSettings.DatabaseName);
+
+        services.AddSingleton(a => {
```

Comment:
Are you sure it should be singleton? Not scoped?

Reply:
When I was reading documentation of Mongo. I see this tip: 
![image](https://github.com/zabila/SportEventOrganizer/assets/17889084/628460cc-7aa1-4245-b127-d94ae518c159)
And decided to use Singleton.